### PR TITLE
Changes to support Node 16

### PR DESCRIPTION
**Pull request by madkoo**
Changes to support Node16

- [x] add V2 for emailReportTask
- [x] V2 should support Node16
- [x] Add pipeline to automate deployment
- restructure, rename manifest file
- update required pacakges
- rename V2 extension
- rename to V1, add new package commands
- restructure extension, add webpack to package application
- resturcure, setup packagin for dev and release
- add initial pipeline to build, package and deploy emailReporting extension

### CODE REVIEW

**Comment by MOlausson** [comment]
@madkoo there's a lot of added intermediary files that I don't think should be committed. Can you clean that up so it's easier to do the review?

**Comment by madkoo** [comment]
@MOlausson , indeed included files that shouldn't have gone in. Cleaned up now.

File: configs/dev.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+    "id": "EmailReportExtension",
+    "name": "Email Report Extension (Dev)",
+    "public": false,
+    "publisher": "solidify-labs"
```
Comment (MOlausson) [inline configs/dev.json:5]:
```suggestion
    "publisher": "ms-devlabs"
```

**Review by MOlausson — APPROVED** [review]
Looks good now.

**Pull request merged**